pom.xml

```diff
@@ -72,7 +72,7 @@
   </mailingLists>
 
   <prerequisites>
-    <maven>3.0</maven>
+    <maven>3.0.5</maven>
```

Comment:
Debian 7's back on maven 3.0.4 but that's pretty ancient so if we do need 3.0.5 then go for it.